linode/volume/framework_schema_resource.go

```diff
@@ -88,6 +89,9 @@ var frameworkResourceSchema = schema.Schema{
 			Description: "The Linode ID where the Volume should be attached.",
 			Optional:    true,
 			Computed:    true,
+			PlanModifiers: []planmodifier.Int64{
+				int64planmodifier.UseStateForUnknown(),
```

Comment:
I was getting some weird infinite diffs & apply errors without this, figured I'd sneak it in here 🙂